src/components/AutoSuggestion/SingleLineCodeEditor.tsx

```diff
@@ -89,11 +88,10 @@ const SingleLineCodeEditor = ({
               editor.getScrollInfo()
           );
         }}
-        onBeforeChange={(_editor, _data, value, next) => {
-            // Remove entered new lines
-            const trimmedValue = value.replace(/[\r\n]/g, "");
-            if (trimmedValue !== value) {
-                _editor.setValue(trimmedValue)
+        onBeforeChange={(_editor, data, value, next) => {
```

Comment:
Should be `_value` if it's not used. Preventing warnings/errors, see `yarn compile`.